src/drive/web/modules/upload/index.js

```diff
@@ -85,22 +95,50 @@ const processNextFile = (
       fileUploadedCallback(uploadedFile)
     }
     dispatch({ type: RECEIVE_UPLOAD_SUCCESS, file })
-  } catch (error) {
-    // eslint-disable-next-line no-console
-    console.warn(error)
-    const statusError = {
-      409: CONFLICT,
-      413: QUOTA
+  } catch (uploadError) {
+    if (uploadError.status === CONFLICT_ERROR) {
+      try {
+        error = uploadError
+        const path = await getFilePath(client, file, dirID)
+        if (
+          flag('handle-conflicts') &&
+          !isShared(sharingState, { path }) &&
+          !hasSharedParent(sharingState, { path })
+        ) {
+          const uploadedFile = await overwriteFile(client, file, path)
+          fileUploadedCallback(uploadedFile)
+          dispatch({ type: RECEIVE_UPLOAD_SUCCESS, file, isUpdate: true })
+          error = null
+        }
```

Comment:
Not blocking but I would prefer to have a kind of `retry` system instead of having to deal with the "same thing" in an `catch` block.

Reply:
I'm not sure I understand what it would look like

Reply:
Something calling itself a few times with new parameters? (In this case juste two times). But forget my comment ;)